### ISSUE TITLE
Formatting issues when pasting in WSL tab

### ISSUE THREAD

**Issue by europaisch** [microsoft/terminal#3502]
# Environment

```none
Windows build number: Microsoft Windows [Version 10.0.19018.1]
Windows Terminal version (if applicable): 0.6.2951.0
WSL version: 2
WSL distribution: Ubuntu
```

# Steps to reproduce

Copy any text with multiple lines into the WSL terminal tab.
Note: this bug does NOT occur in the command line and PowerShell tabs.

# Expected behavior

Same amount of lines as in the source and the paste not affecting existing text in unexpected ways.

# Actual behavior

I get extra lines in-between regardless of whether the source I copy from is LF or CRLF when I paste it into WSL in the terminal.
![image](https://user-images.githubusercontent.com/47948375/68528431-512a6080-02fb-11ea-994a-970cf99170ff.png)
![image](https://user-images.githubusercontent.com/47948375/68528437-5edfe600-02fb-11ea-87b4-156d58460d64.png)

Existing text is also affected.

before:
![image](https://user-images.githubusercontent.com/47948375/68528480-e2013c00-02fb-11ea-9ab1-1c5239438602.png)
after:
![image](https://user-images.githubusercontent.com/47948375/68528484-f5aca280-02fb-11ea-9e56-c385bf8a345d.png)

another example:

before:
![image](https://user-images.githubusercontent.com/47948375/68528500-1e349c80-02fc-11ea-9be3-855ae2e2c397.png)
after:
![image](https://user-images.githubusercontent.com/47948375/68528508-36a4b700-02fc-11ea-9b79-728b8fe24413.png)

**Comment by d-bingham** [comment]
This is, as far as I can tell, a duplicate of #1091 and was fixed in #3449.

I just checked this with the current build (specifically with nano, as I don't actually use it as part of my workflow, and its handling of some terminal settings is different than, say, vi.  Also I've been doing development for a _long_ time and that causes paranoia), and it looks like it's working.

So it's working in the live build off the repository, and it'll be in 0.7.
